tests/component/test_write_properties.py

```diff
@@ -0,0 +1,168 @@
+"""Tests for validating write properties."""
+
+import pytest
+
+import nidaqmx
+import nidaqmx.system
+from nidaqmx.constants import WriteRelativeTo
+from nidaqmx.errors import DaqError, DAQmxErrors
+
+
+def test__write__get_int32_property__returns_value(any_x_series_device):
+    """Test to validate getter for int32 property."""
+    with nidaqmx.Task() as write_task:
+        write_task.ao_channels.add_ao_voltage_chan(
+            any_x_series_device.ao_physical_chans[0].name, max_val=10, min_val=-10
+        )
```

Comment:
Create local fixture for duplicate codes

Please check the comments in https://github.com/ni/nidaqmx-python/pull/250.

Reply:
Local fixture created.